File: 실습/3장/06_pair_destructuring/Main.java
public class Main {

    static class Pair {
        private final String language;
        private final int year;

        public Pair(String language, int year) {
            this.language = language;
            this.year = year;
        }

        public String getLanguage() {
            return language;
        }

        public int getYear() {
            return year;
        }
    }

    public static void main(String[] args) {
        int result = 10 * 20;
        System.out.println(result);

        Pair pair = new Pair("Java", 2024);

        System.out.println(pair.getLanguage());
        System.out.println(pair.getYear());
    }
}

File: 실습/3장/06_pair_destructuring/Main.kt
infix fun Int.times(value: Int): Int {
    return this * value
}

fun main() {
    val result = 10 times 20
    println(result)

    val pair = "Kotlin" to 2024
    val (language, year) = pair

    println(language)
    println(year)
}